src/utils/getDescription.ts

```diff
@@ -0,0 +1,7 @@
+export const getDescription = (
+  description: string | { value: string } | undefined
+): string => {
```

Comment:
```suggestion
) => {
```
в таких простых функциях TS сам тип выведет, можно не указывать